aiohttp_client_cache/backends/gridfs.py

```diff
@@ -69,7 +70,9 @@ async def size(self) -> int:
         return self.db['fs.files'].count()
 
     # TODO
-    async def values(self) -> Iterable[ResponseOrKey]:
+    async def values(self) -> AsyncIterable[ResponseOrKey]:
+        if False:
+            yield
```

Comment:
heh. was gonna ask for a comment here but I suppose the TODO is enough of an ACK :)

Reply:
Yeah, it's stupid, but without a `yield`, it doesn't get typed correctly as an `AsyncIterable`.